flint/imager/wsclean.py

```diff
@@ -882,34 +887,67 @@ def create_wsclean_cmd(
     )
 
 
-def rotate_cube(output_cube_name) -> None:
-    logger.info(f"Rotating {output_cube_name=}")
-    # This changes the output cube to a shape of (chan, pol, dec, ra)
-    # which is what yandasoft linmos tasks like
-    with fits.open(output_cube_name, mode="update", memmap=True) as hdulist:
-        hdu = hdulist[0]
-        new_header = hdu.header
-        data_cube = hdu.data
-
-        tmp_header = new_header.copy()
-        # Need to swap NAXIS 3 and 4 to make LINMOS happy - booo
-        for a, b in ((3, 4), (4, 3)):
-            new_header[f"CTYPE{a}"] = tmp_header[f"CTYPE{b}"]
-            new_header[f"CRPIX{a}"] = tmp_header[f"CRPIX{b}"]
-            new_header[f"CRVAL{a}"] = tmp_header[f"CRVAL{b}"]
-            new_header[f"CDELT{a}"] = tmp_header[f"CDELT{b}"]
-            new_header[f"CUNIT{a}"] = tmp_header[f"CUNIT{b}"]
-
-        # Cube is currently STOKES, FREQ, RA, DEC - needs to be FREQ, STOKES, RA, DEC
-        data_cube = np.moveaxis(data_cube, 1, 0)
-        hdu.data = data_cube
-        hdu.header = new_header
-        hdulist.flush()
+def rotate_cube(output_cube_name: str, inplace: bool = True) -> None:
```

Comment:
I also forgot our own rule of returning a value. We should probably return the `Path` object on success. That also means we can return early on given logical branches